packages/flutter_tools/test/integration.shard/test_test.dart

```diff
@@ -313,6 +313,8 @@ Future<ProcessResult> _runFlutterTest(
     '--no-color',
     '--no-version-check',
     '--no-pub',
+    '--reporter',
+    'compact',
     ...extraArguments,
```

Comment:
Note that this is likely a hack solution to allow tests to pass on windows – need to determine why tests are failing on windows when `reporter=compact` is not explicitly being passed in

cc @christopherfujino (just tagging so I know where we left off today)

Reply:
hmm, if this fixes it, then what is it defaulting to when it fails?

Reply:
Okay I did some digging and this is what I found, I believe the default behavior will actually be based on the platform (windows vs linux/macos)

There is a variable that `test_core` uses called `reporter` which has a `defaultReporter` backup if one is not provided within the `Configuration` class:
https://github.com/dart-lang/test/blob/master/pkgs/test_core/lib/src/runner/configuration.dart#L94-L96

And when we don't pass an option for the `reporter`, you can see the logic for how `defaultReporter` is determined here:
https://github.com/dart-lang/test/blob/master/pkgs/test_core/lib/src/runner/configuration/reporters.dart#L60-L67

This will depend on the `canUseSpecialChars` boolean which is defined by this code snippet below, and I believe this boolean resolves differently on windows which results in the error

```dart
/// Whether or not the current terminal supports ansi escape codes.
///
/// Otherwise only printable ASCII characters should be used.
bool get canUseSpecialChars =>
    (!Platform.isWindows || stdout.supportsAnsiEscapes) && !inTestTests;
```
https://github.com/dart-lang/test/blob/master/pkgs/test_core/lib/src/util/io.dart#L84-L88